recipes/diligent-core/all/conandata.yml

```diff
@@ -15,18 +15,10 @@ patches:
   "api.252003":
     - patch_file: "patches/0019-252003-exclude-tests.patch"
       base_path: "source_subfolder"
-    - patch_file: "patches/0020-252003-dont-install-3d-party-license.patch"
-      base_path: "source_subfolder"
-    - patch_file: "patches/0021-252003-fix-glslang-usage.patch"
-      base_path: "source_subfolder"
     - patch_file: "patches/0022-252003-fix-glslang-include.patch"
       base_path: "source_subfolder"
     - patch_file: "patches/0023-252003-fix-warning-as-error.patch"
       base_path: "source_subfolder"
```

Comment:
This patch should also not be necessary as you set DILIGENT_CLANG_COMPILE_OPTIONS and DILIGENT_MSVC_COMPILE_OPTIONS

Reply:
I still have an error when I delete a patch:
```
[ 38%] Building CXX object source_subfolder/Graphics/ShaderTools/CMakeFiles/Diligent-ShaderTools.dir/src/SPIRVTools.cpp.o
/home/andrei/.conan/data/diligent-core/api.252003/andrei/test/build/6881f80c17218d135fdf22c03d23b300c1292b03/source_subfolder/Graphics/ShaderTools/src/GLSLangUtils.cpp:36:13: error: 'ENABLE_HLSL' macro redefined [-Werror,-Wmacro-redefined]
#    define ENABLE_HLSL
            ^
<command line>:7:9: note: previous definition is here
#define ENABLE_HLSL 1
        ^
1 error generated.
```

Reply:
With https://github.com/DiligentGraphics/DiligentCore/commit/43ae908b68c3fe140e0caebd02b7bf04311c211e it should work now

Reply:
Great, waiting for new tag to produce a patchesless version

Reply:
Do you want to remove this patch now?

Reply:
The latest diligent - core version does not have any patches already